source/main/modules/export/index.ts

```diff
@@ -96,7 +96,7 @@ export default async function makeExport (options: ExporterOptions): Promise<Exp
     )
   } else {
     // Run Pandoc
-    const defaultsFile = await writeDefaults(options.format, options.sourceFile, options.targetFile)
+    const defaultsFile = await writeDefaults(options.format, options.file.frontmatter, options.sourceFile, options.targetFile)
```

Comment:
Just pass all file frontmatters (e.g. `options.files.map(file => file.frontmatter)`) and merge them?